bubblegum/xrf/view/guessparam.enaml

```diff
@@ -0,0 +1,392 @@
+                                pb7.text = 'default value'
+                            else:
+                                fd7.value = param_m.param_d_perm['fwhm_offset']['value']
+                                fd7.enabled = False
+                                pb7.text = 'user value'
+
+                    Label: lbl8:
+                        text = 'fwhm Coef, b2'
+                        tool_tip = 'width**2 = (b1/2.3548)**2 + 3.85*b2*E, 3.85keV is electron-hole pair creation energy in silicon '
+                    FloatField: fd8:
+                        enabled = False
+                        value := param_m.param_d['fwhm_fanoprime']['value']
+                        maximum_size = 100
+                    PushButton: pb8:
+                        text = 'user value'
+                        checkable = True
+                        clicked ::
+                            if checked:
+                                fd8.enabled = True
+                                pb8.text = 'default value'
+                            else:
+                                fd8.value = param_m.param_d_perm['fwhm_fanoprime']['value']
+                                fd8.enabled = False
+                                pb8.text = 'user value'
+
+                    Label: lbl9:
+                        text = 'Compton fwhm Coef'
+                    FloatField: fd9:
+                        enabled = False
+                        value := param_m.param_d['compton_fwhm_corr']['value']
+                        maximum_size = 100
+                    PushButton: pb9:
+                        text = 'user value'
+                        checkable = True
+                        clicked ::
+                            if checked:
+                                fd9.enabled = True
+                                pb9.text = 'default value'
+                            else:
+                                fd9.value = param_m.param_d_perm['compton_fwhm_corr']['value']
+                                fd9.enabled = False
+                                pb9.text = 'user value'
+
+        PushButton: pb10:
+            text = 'Save'
+            clicked ::
+                pass
+                #param_m.save_param(param_m.param_d)
+        PushButton: pb11:
+            text = 'Save As'
+            clicked ::
+                pass
+        PushButton: pb12:
+            text = 'Find Peaks'
+            #status_tip = 'Not Running.'
+            clicked ::
+                param_m.set_data(fileio_m.data)
+                param_m.find_peak()
+                param_m.arange_prefit_result()
+                status_tip = 'Running'
+                if (len(param_m.total_y) == 0):
+                    #lb15.text = 'No auto fit result is available.'
+                    pb13.enabled = False
+                    pb14.enabled = False
+
+                else:
+                    #lb15.text = 'Auto fit is finished.'
```

Comment:
Do you want to remove all lines that are commented in here? There are quite a number